docker-compose.yml

```diff
@@ -2,9 +2,7 @@ version: "3"
 
 services:
   postgres:
-    build:
-      context: ./docker/
-      dockerfile: ./plv8.dockerfile
+    image: bryanchriswhite/ledger-subquery-postgres:latest
```

Comment:
We should really have control of our own images to be honest. This looks like a merge thing but we should be able to re-create all the pieces from this repo